src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -661,10 +661,18 @@ def load_arguments(self, _):
                             'Creates a new record set if one does not exist.')
             c.argument('if_none_match', help='Create the record set only if it does not already exist.',
                        action='store_true')
+            if item == 'srv':
+                c.argument('priority', type=int)
+                c.argument('weight', type=int)
+                c.argument('port', type=int)
 
         with self.argument_context('network dns record-set {} remove-record'.format(item)) as c:
             c.argument('record_set_name', options_list=['--record-set-name', '-n'], help='The name of the record set relative to the zone.')
             c.argument('keep_empty_record_set', action='store_true', help='Keep the empty record set if the last record is removed.')
+            if item == 'srv':
+                c.argument('priority', type=int)
+                c.argument('weight', type=int)
+                c.argument('port', type=int)
```

Comment:
You should update on the following existing lines https://github.com/Azure/azure-cli/blob/4bed2d2778f974b432ea5f2f1cd470d1897cfa7e/src/azure-cli/azure/cli/command_modules/network/_params.py#L725-L729